babel.config.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   presets: ['module:metro-react-native-babel-preset'],
   plugins: [
+    'react-native-reanimated/plugin',
```

Comment:
I don't see react-native-reanimated in package json. Either remove that or add it to package.json